src/command_line_parser.cc

```diff
@@ -1707,7 +1718,7 @@ CLParser::ParseCommandLine(int argc, char** argv)
           break;
         }
         case long_option_idx_base + 62: {
-          if (std::stoi(optarg) < 0) {
+          if (std::stoi(optarg) <= 0) {
```

Comment:
This was a random "bug" fix.

As the error message below says, user should not be able to specify request count of zero. It's means no benchmark gets run, which is useless.